src/libs/Clipboard/index.js

```diff
@@ -62,7 +62,8 @@ function setHTMLSync(html, text) {
 
     if (isComposer) {
         firstAnchorChild.setSelectionRange(originalSelection.start, originalSelection.end, originalSelection.direction);
-    } else {
+    } else if (originalSelection.anchorNode && originalSelection.focusNode) {
+        // We are adding a check for anchorNode and focusNode to prevent null values from being passed to setBaseAndExtent as per recent changes in Safari
```

Comment:
If we are to add a comment, I think we should focus on:

- why these values can become `null` in practice
- that `null` values are disallowed by this API per the standard

The fact that Safari actually started enforcing that is the least important matter and the comment might get outdated once other implementors start enforcing these `null`-checks